### DISCUSSION TITLE
generate removes go.mod entries after generating models and resolvers

### DISCUSSION THREAD

**Discussion by catielanier** [99designs/gqlgen#2005] [Q&A]
### What happened?

- After running `go get github.com/99designs/gqlgen generate`, the following entries from `go.mod` are removed, forcing me to run `go get` to fetch them every time I generate:
  - `github.com/99designs/gqlgen/internal/imports`
  - `github.com/99designs/gqlgen/codegen/config`
  - `github.com/99designs/gqlgen/cmd`

### What did you expect?
- The packages added to `go.mod` after running these three `go get` commands should remain in `go.mod` after running `go get github.com/99designs/gqlgen generate`. It's annoyingly tedious to have to run three extra commands every time I want to generate.

### Minimal graphql.schema and models to reproduce
- Can use default schema to reproduce.

### versions
 - `gqlgen version`? 0.16.0
 - `go version`? 1.17.7
 - dep or go modules? go

**Comment by frederikhors** [comment]
I cannot reproduce this.

**Comment by catielanier** [comment]
Please see included video to corroborate issue's produceability.

https://youtu.be/453vXRwQOVI

**Comment by ghost** [comment]
Happens also here.

gqlgen version? 0.17.2
go version? 1.18 / 1.17.7

Thanks

**Reply by catielanier** [reply]
Thanks for corroborating this. I had to write a simple npm script to have it do the entire install flow before it runs, so that I don't have to manually enter the commands every time.

Otherwise this would be unbearable for me.

**Reply by frederikhors** [reply]
But why are you launching the command `go get github.com/99designs/gqlgen generate`?

**Reply by ghost** [reply]
Running `go run github.com/99designs/gqlgen generate`

**Comment by catielanier** [comment]
I’m running go run github.com/99designs/gqlgen generate

**Comment by ghost** [comment]
Adding gqlgen to tools.go as described in the README solved it for me. Thanks
